level06/tests/Test.hs

```diff
@@ -1,9 +1,9 @@
 {-# LANGUAGE OverloadedStrings #-}
 module Main where
 
-import           Control.Monad  (void, join)
+import           Control.Monad  (join, void)
 
-import Data.Bifunctor (first)
+import           Data.Bifunctor (first)
```

Comment:
It might be worth calling out Data.Bifunctor either here or in the README so people go and have a look (and hopefully ask questions)